jackson/src/main/java/com/spotify/hamcrest/jackson/IsJsonBoolean.java

```diff
@@ -58,14 +58,14 @@ protected boolean matchesNode(BooleanNode node, Description mismatchDescription)
     if (booleanMatcher.matches(value)) {
       return true;
     } else {
-      mismatchDescription.appendText("was a boolean node ");
+      mismatchDescription.appendText("was a boolean node with value that ");
       booleanMatcher.describeMismatch(value, mismatchDescription);
       return false;
     }
   }
 
   @Override
   public void describeTo(Description description) {
-    description.appendText("a boolean node ").appendDescriptionOf(booleanMatcher);
+    description.appendText("a boolean node with value that ").appendDescriptionOf(booleanMatcher);
```

Comment:
I think this is OK. The user can add an `is()` to get `a boolean node with value that is ANYTHING`.

e.g.

```java
    assertThat(NF.booleanNode(false), is(not(jsonBoolean(is(anything())))));
    // Expected: is not a boolean node with value that is ANYTHING
     // but: was <false>

    assertThat(NF.booleanNode(false), is(jsonBoolean(is(equalTo(true)))));
// Expected: is a boolean node with value that is <true>
     // but: was a boolean node with value that was <false>
```